spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -647,9 +639,11 @@
               expected_wait_until = 1.hour.from_now
               expect do
                 job.perform(Time.zone.now)
+                pending_enrollment.reload
```

Comment:
this is a lot of reload.... are they strictly needed? could we add them in more specific places like `pending_enrollment.reload.proofed_at` below?

Reply:
In the spec file, there is a maximum of 5 records, while there are a lot of tests the number of reloads shouldn't have too large an impact.